guides/v2.3/graphql/functional-testing.md

```diff
@@ -336,3 +336,113 @@ Use the following functions to cover expected exceptions:
 - `expectExceptionMessage`
 - `expectExceptionMessageRegExp`
 - `expectExceptionObject`
+
+## Run functional tests
+
+### Configure your instance
+
+1. Copy `phpunit_graphql.xml.dist` file to `phpunit_graphql.xml` in the `dev/tests/api-functional/phpunit.xml` folder.
+
+2. Open for edit `phpunit_graphql.xml` and set values for the TESTS_BASE_URL, TESTS_WEBSERVICE_USER, TESTS_WEBSERVICE_APIKEY options:
+
+    ```xml
+    ...
+    <!-- Webserver URL -->
+    <const name="TESTS_BASE_URL" value="http://magento.url"/>
+    <!-- Webserver API user -->
+    <const name="TESTS_WEBSERVICE_USER" value="admin"/>
+    <!-- Webserver API key -->
+    <const name="TESTS_WEBSERVICE_APIKEY" value="123123q"/>
+    ...
+    ```    
+
+3. Change current directory to `dev/tests/api-functional`:
+
+   ```bash
+   cd <magento_root>/dev/tests/api-functional
+   ```
+
+### Run all tests of API functional test 
+
+**Syntax**
+
+```bash
+../../../vendor/bin/phpunit testsuite/<Vendor>/<Module>/<TestFile>.php
```

Comment:
Just a recommendation. We provide examples assuming that we in the `<magento_root>/dev/tests/api-functional` directory as you have mentioned before. However, readers might skip this part and go straight to the examples at the end of the article. From this point we have a chance for confusion since without changing directory to the `<magento_root>/dev/tests/api-functional` the provided examples don't work. 
To avoid such situations I would recommend refactoring all examples to run everything from `<magento_root>` directory. Example:

```bash
vendor/bin/phpunit -c dev/tests/api-functional/phpunit_graphql.xml dev/tests/api-functional/testsuite/<Vendor>/<Module>/<TestFile>.php
```

Reply:
In this case , readers may skip points 1 and 2 :)

**fixed**